src/vcpkg/cgroup-parser.cpp

```diff
@@ -28,7 +28,7 @@ namespace vcpkg
         using P = ParserBase;
         constexpr auto is_separator_or_lineend = [](auto ch) { return ch == ':' || P::is_lineend(ch); };
 
-        auto parser = ParserBase(text, origin);
+        ParserBase parser{text, origin};
```

Comment:
Note that this does not actually remove any copies. (I'm happy with it though :) )

Reply:
This is just a stylistic thing. IMO using auto when declaring a variable adds (a little) obscurity.

Reply:
Stylistic, yes. I prefer the original style.

Reply:
> Stylistic, yes. I prefer the original style.

Why do you prefer the original style? I don't like it because it is longer.